src/backend/postmaster/postmaster.c

```diff
@@ -5676,6 +5741,21 @@ dummy_handler(SIGNAL_ARGS)
 {
 }
 
+/*
+ * Terminate stopped background writer process (should be invoked only
+ * during test).
+ */
+static void
+TerminateBgWriterHandler()
```

Comment:
Maybe this should be but behind `#ifdef FAULT_INJECTOR`?

Reply:
> Maybe this should be but behind #ifdef FAULT_INJECTOR?

The same comment as above.